controllers/PageController.php

```diff
@@ -457,4 +458,77 @@ public function actionToggleNumbering(int $id)
             return $this->redirect(Url::previous());
         }  
     }
+
+    public function actionMerge(int $id) {
+
+        $page = $this->getWikiPage($id);
+
+        if (!$page) {
+            throw new HttpException(404, 'Wiki page not found!');
+        }
+
+        $form = (new PageEditForm(['container' => $this->contentContainer]))->forPage($id);
+
+        if (!$form->load(Yii::$app->request->post())) {
+            throw new HttpException(404);
+        }
+
+        $submittedRevision = new WikiPageRevision();
+        $submittedRevision->revision = time();
+        $submittedRevision->content = $form->revision->content;
+        $submittedRevision->isCurrentlyEditing = true;
+
+        $mergedRevision = $page->createRevision();
+        $mergedRevision->content = $page->latestRevision->content.' '.$submittedRevision->content;
```

Comment:
There should be some kind of separation in between. At least a new line.
But maybe you can add also a notice, e.g., "Conflicting changes from xxx ..." with a seperation line.